### PR TITLE
Update redis.py

### PR DESCRIPTION
**Pull request by akshayghatiki311**
Provided an option to connect to redis cloud using password,port and db along with existing parameters

### CODE REVIEW

File: langchain/memory/chat_message_histories/redis.py [hunk 1]
```diff
@@ -21,6 +21,9 @@ def __init__(
         url: str = "redis://localhost:6379/0",
         key_prefix: str = "message_store:",
         ttl: Optional[int] = None,
+        db: int | None=...,
+        password: str | None=...,
```
Comment (Spartee) [inline langchain/memory/chat_message_histories/redis.py:25]: This should just be ``**kwargs`` and passed to the ``Redis.from_url`` method as such. There are a number of other things a user might want to pass like TLS settings. Either way, the username, password, and port should always just be passed in the url ``redis://username:password@host:port``

**Review by Spartee — CHANGES_REQUESTED** [review]
Change to kwargs.

**Comment by leo-gan** [comment]
@akshayghatiki311 Hi , could you, please, resolve the merging issues? After that ping me and I push this PR for the review. Thanks!

**Comment by efriis** [comment]
Closing because the PR wouldn't line up with the current directory structure of the library (would need to be in /libs/langchain/langchain instead of /langchain). Feel free to reopen against the current head if it's still relevant!

**Pull request closed without merging**